src/header.tsx

```diff
@@ -178,6 +178,10 @@ export const HeaderLayout: React.FC<HeaderLayoutProps> = ({
 }) => {
     const {isTabletOrSmaller} = useScreenSize();
 
+    const isBleedActivated = (): boolean => {
+        return bleed && isInverse;
+    };
```

Comment:
this can be a const, no need a function.

Reply:
Sure, fixed!